### ISSUE TITLE
NH miner alway require Internet connection ?!!

### ISSUE THREAD

**Issue by dualshock2** [nicehash/NiceHashMiner-Archived#16]
I can't run Nicehash Miner.  It's always laggy and show this notify:

http://pik.vn/2015d4bf4f5d-c948-4c60-978e-de8a68f4515c.png

After I press "Yes", it laggy for long time and re-show this notify.
If press "No", it quit application. 

I try all statrum: USA, EU, HK, JP...  run as Administrator..... but its same error !

My rig: 
Windows 7 x64bit.
4G Ram.
4 x 7950.
Driver 15.7.1
No firewall, no anti-virus. 

I can run from **runme.bat** and everything work fine. But cannot run Nicehash Miner UI. 
Pls help me ! Thankyou !
![error](https://cloud.githubusercontent.com/assets/16477528/12035942/09118380-ae74-11e5-9969-3b4b0bae25a6.png)

**Comment by theLosers106** [comment]
Could you send the log.txt file to nhm@nicehash.com .. That would be helpful

**Comment by dualshock2** [comment]
Thankyou for reply ! 
I re-install Windows  10 and everythings work fine !

**Comment by NogaAllTK** [comment]
I reinstalled Windows 10 and it didnt work. I have the same problem as you did.

**Comment by NaitorStudios** [comment]
Here's how to fix it:
I've found out that NiceHash Miner (and some other programs) can't detect internet connection when IPv4 and IPv6 are enabled at the same time.
Disable one of them and the problem should go away. I think IPv6 is better, so I let it enabled and disabled IPv4, but I don't know if there's any drawbacks.

To disable just right click the network tray icon, then click Open Network and Sharing Center.
In the window that show up, look for "Connections: " right below HomeGroup, then click whats on the right side (should be your network name).
New window, Properties, you will see a list, check or uncheck IPv6 or IPv4, you must let at least one checked.

For the developers:
NiceHash Miner can't determine with IP version to use when both are enabled, hope in a future update it get fixed so we can use both at same time and people won't freak out with this issue.

EDIT: 
If this helped you, you may consider a donation, here's my Bitcoin Address:
1KZZX3xfo3PoavhWvVZJPjETTbCprwqBQa

I've recorded an improvised how to video:
https://www.youtube.com/watch?v=fnZ-eoGWt74

**Comment by NogaAllTK** [comment]
Thank you so very much FuurioBR you helped me when even the support couldn't!!!!!! For me it worked with only IPv4 on. You are awesome! Send me your receive address so I can reward you please!

**Comment by NaitorStudios** [comment]
Your welcome, NogaAllTK, I suffered for quite some time with this bug, I know how frustrating it can be.
I noticed that other softwares as affected by this issue, giving other kinds of errors (script errors) or not detecting internet too.
I really looked everything possible to find what was causing this problem, even reinstalled Windows (with did take a lot of time and effort, since I have to reinstall a lot of programs and plugins...).
I hope you get very luck from now, and have a Nice mining, I'm really happy to help! ;)

And for rewards / donations...
BTC address: 1Kz3N3aAnhHcQs2t79bDPQTXgL2DgjFJCg
Oh, if you wanna see my mining stats, I'm kinda of new in it (other address, but same wallet)
https://www.nicehash.com/index.jsp?p=miners&addr=1KZZX3xfo3PoavhWvVZJPjETTbCprwqBQa
